publishers/community/slack/slack_layout.py

```diff
@@ -25,6 +25,7 @@
 BABU = '#00d1c1'
 LIMA = '#8ce071'
 HACKBERRY = '#7b0051'
+BEACH = '#ffb400'
```

Comment:
What does this constant mean? Is it some special characters needs to be skipped or special processed? Can you add a comment?

Reply:
It's an airbnb brand color that I added... since it's not being used I can get rid of it